autotest/gdrivers/rda.py

```diff
@@ -691,6 +691,274 @@ def rda_graph_nominal():
 
     gdal.RmdirRecursive('/vsimem/cache_dir')
 
+    # Test 502
+    handler = webserver.SequentialHandler()
+    handler.add('POST', '/auth_url', 200, {}, '{"access_token": "token", "expires_in": 3600}')
+    handler.add('GET', '/rda_api/metadata/foo/bar/metadata.json', 200, {}, json.dumps(metadata_json))
+    handler.add('GET', '/rda_api/tile/foo/bar/0/0.tif', 502, {}, json.dumps({"Error": "502"}))
+
+    try:
+        org_use_exceptions = gdal.GetUseExceptions()
+        if not org_use_exceptions:
+            gdal.UseExceptions()
+        gdal.PushErrorHandler('CPLQuietErrorHandler')
```

Comment:
complicated way of silencing errors. You can now use
```
with gdaltest.error_handler():
       some code that might output GDAL errors / warnings
```

Reply:
Done